ads/imedia.md

```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# IM.cz
+
+## Example
+
+```html
+<amp-ad width=300 height=250
+	type="imedia"
+	data-id="p1"
+    data-positions='[
+      {"id":"p1", "zoneId":"seznam.novinky.ikona2"},
+      {"id":"p2", "zoneId":"seznam.novinky.ikona"}
+      ]'
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see [ad network documentation](https://iimedia.sbeta.cz/html/im3light/).
+
+Supported parameters:
+
+- width
```

Comment:
The supported params are same for all amp-ad, already document in amp-ad doc, not needed here.